frontend/src/metabase/api/tags/utils.ts

```diff
@@ -47,6 +47,10 @@ export function listTag(type: TagType): TagDescription<TagType> {
   return { type, id: "LIST" };
 }
 
+export function partialListTag(type: TagType): TagDescription<TagType> {
+  return { type, id: "PARTIAL-LIST" };
```

Comment:
After reading [the docs on invalidation tags](https://redux-toolkit.js.org/rtk-query/usage/automated-refetching#advanced-invalidation-with-abstract-tag-ids), I don't think this provides any value over just using `LIST`.

In fact, adding `PARTIAL-LIST` next to list is risky since we might now run the risk of invalidating one but not the other.